utils/loggingcpp/ErrorMessage.txt

```diff
@@ -100,6 +100,10 @@
 
 2053	ERR_FUNC_OUT_OF_RANGE_RESULT	The result is out of range for function %1% using value(s): %2% %3%
 
+2054	ERR_DISKAGG_ERROR	Unknown error while aggregation.
```

Comment:
There is no way to detect which method throws this error. The thow should log at least a method name.